popsycle/synthetic.py

```diff
@@ -133,8 +147,8 @@ def write_galaxia_params(output_root,
     -------------------
     seed : int
          Seed Galaxia will use to generate objects. If not set, script will
-         generate a random number from 0 to 100. This may want to be set for
-         reproducibility.
+         generate a random number from 0 to 100. Setting this seed guarantees
```

Comment:
Standard procedure is to grab the system time for the random seed.... I think you could still do this (check online; but maybe just call seed with None?).

Reply:
Galaxia requires a two digit number from 0 to 100. This number will now be drawn from the current time.
```
if seed is None:
        # Grab last two digits of the current Epochal time
        seed = int(str(time.time())[-2:])
```